sample/test/fails/test.yaml

```diff
@@ -0,0 +1,13 @@
+test:
+    attribute: ignore
+    tests:
+        - fail-output-1:
+            syspage: fail_output_1
+            harness: test-fail-output-1.py
+        - fail-output-2:
+            syspage: fail_output_2
+            harness: test-fail-output-2.py
+        - fail-unity:
```

Comment:
please add test which will fail with exception (eg NULL dereference) and think about how to process it. Might be done in the separate PR to speed up merging of this one.

(i think it would be enough to have full output of every failed test)

Reply:
This will require parsing output buffer in case of `FAIL_TIMEOUT` for exception pattern. Will be done in separate PR.

Reply:
I don't care about automatically parsing the exception buffer, I need to see the full output of the failed testcase execution, eg:
![Screenshot_20210406_164447](https://user-images.githubusercontent.com/3939983/113729558-69243080-96f7-11eb-905b-9b63b8a13d91.png)

Reply:
I added printing output buffer and pattern that we were looking for.